src/help/help.go

```diff
@@ -64,6 +68,103 @@ func help(topic string) string {
 	if f, present := m[topic]; present {
 		return helpFromBuildRule(f.FuncDef)
 	}
+
+	// Check plugins
+	if pluginHelp := helpForPlugin(topic); pluginHelp != "" {
+		return pluginHelp
+	}
+
+	return ""
+}
+
+func helpForPlugin(topic string) string {
+	config, err := core.ReadDefaultConfigFiles(nil)
+	if err != nil {
+		panic("Failed to read config")
+	}
+	if _, ok := config.Plugin[topic]; ok {
+		message := fmt.Sprintf("${BOLD_BLUE}%v${RESET} is a plugin defined in the .plzconfig file.", topic)
+		if val, ok := config.Plugin[topic].ExtraValues["subrepo"]; ok {
+			subrepo := val[0]
+			buildLabel := core.BuildLabel{PackageName: "", Name: subrepo, Subrepo: subrepo}
+			state := newState()
+
+			// Parse the subrepo (Run reads the plugin config into config)
+			plz.Run([]core.BuildLabel{buildLabel}, nil, state, config, state.TargetArch)
+			downloadBuildLabel := core.BuildLabel{PackageName: "", Name: "_" + subrepo + "#download", Subrepo: ""}
+			if t := state.Graph.Target(downloadBuildLabel); t != nil {
+				if urls := t.AllURLs(state); len(urls) == 1 {
+					message += fmt.Sprintf(" It's loaded from %v\n", urls[0])
+				} else {
+					message += "\n"
+				}
+			} else {
+				message += "\n"
+			}
+
+			state = state.Graph.Subrepo(subrepo).State
+			config := state.Config
+			if config.PluginDefinition.Description != "" {
+				message += "\n" + config.PluginDefinition.Description + "\n"
+			}
+			if config.PluginDefinition.Documentation != "" {
+				message += "\n" + config.PluginDefinition.Documentation + "\n"
+			}
+			configOptions := ""
+			for _, v := range config.PluginConfig {
```

Comment:
Might be nice to handle repeated values here, and change `str` to `string` to be inline with the please website:
https://please.build/config.html#parse.buildfilename